### ISSUE TITLE
SyntaxError: Unexpected token 'export'

### ISSUE THREAD

**Issue by eden-lane** [chempogonzalez/vite-plugin-svgr-component#1]
Error while trying to start dev server:
```
failed to load config from /Users/eden_lane/Projects/work/create-rmr-app/src/templates/init/vite.config.js
error when starting dev server:
/Users/eden_lane/Projects/work/create-rmr-app/src/templates/init/node_modules/vite-plugin-svgr-component/lib/index.js:1
export { svgrComponent, svgrComponent as default } from './src';
^^^^^^
```

My config is:
```js
import {defineConfig} from "vite";
import VitePluginLinaria from 'vite-plugin-linaria';
import SvgrComponentPlugin from 'vite-plugin-svgr-component';

export default defineConfig({
  root: './src',
  plugins: [
    VitePluginLinaria(),
    SvgrComponentPlugin(),
  ]
})

```

**Comment by chempogonzalez** [comment]
Hi! Thanks for the issue!. Sorry for the delayed response.

I have published a new version with this issue fixed and for now the package is stable. I have added a test-app folder with a new vite app to ensure all is working fine including typings.

Thanks again!

**Comment by chempogonzalez** [comment]
This was solved in the following commit [#72160c15763fc3a53d0a5f09441ec14cb8e6a080](https://github.com/chempogonzalez/vite-plugin-svgr-component/commit/72160c15763fc3a53d0a5f09441ec14cb8e6a080)
